modules/dynamic-provisioning-azure-disk-definition.adoc

```diff
@@ -9,35 +9,35 @@
 .azure-advanced-disk-storageclass.yaml
 [source,yaml]
 ----
-kind: StorageClass
 apiVersion: storage.k8s.io/v1
+kind: StorageClass
 metadata:
-  name: slow
+  name: managed-premium
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
 provisioner: kubernetes.io/azure-disk
+volumeBindingMode: WaitForFirstConsumer <1>
+allowVolumeExpansion: true
 parameters:
-  storageAccount: azure_storage_account_name  <1>
-  storageaccounttype: Standard_LRS  <2>
-  kind: Dedicated  <3>
+  kind: Managed <2>
+  storageaccounttype: Premium_LRS <3>
+reclaimPolicy: Delete
 ----
-<1> Azure storage account name. This must reside in the same resource
-group as the cluster. If a storage account is specified, the `location`
-is ignored. If a storage account is not specified, a new storage
-account gets created in the same resource group as the cluster. If you
-are specifying a `storageAccount`, the value for `kind` must be `Dedicated`.
-<2> Azure storage account SKU tier. Default is empty. Note that Premium
-VMs can attach both `Standard_LRS` and `Premium_LRS` disks, Standard VMs
-can only attach `Standard_LRS` disks, Managed VMs can only attach
-managed disks, and unmanaged VMs can only attach unmanaged disks.
-<3> Possible values are `Shared` (default), `Dedicated`, and `Managed`.
+<1> Using `WaitForFirstConsumer` is strongly recommended. This provisions the volume while allowing enough storage to schedule the pod on a free worker node from an available zone.
+<2> Possible values are `Shared` (default), `Managed`, and `Dedicated`.
++
+[IMPORTANT]
+====
+Red Hat only supports the use of `kind: Managed` in the storage class.
+
+With `Shared` and `Dedicated`, Azure creates unmanaged disks, while {product-title} creates a managed disk for machine OS (root) disks. But because Azure Disk does not allow the use of both managed and unmanaged disks on a node, unmanaged disks created with `Shared` or `Dedicated` cannot be attached to {product-title} nodes.
+====
+
+<3> Azure storage account SKU tier. Default is empty. Note that Premium VMs can attach both `Standard_LRS` and `Premium_LRS` disks, Standard VMs can only attach `Standard_LRS` disks, Managed VMs can only attach managed disks, and unmanaged VMs can only attach unmanaged disks.
```

Comment:
General note on callouts: for this third one, the item is defined ("Azure storage account SKU tier.") and then the default and other info is given. Should the first two also start with a brief mention of what they are?

Reply:
It's a valid question. I see a mix in general for callouts. In this case, I think it defines it because this callout is referring to the SKU tier (`Premium_LRS` or `Standard_LRS`). The others are calling out the recommended formatting or parameter types. In other words, there's not a standard? ¯\_(ツ)_/¯ 
I'll leave it as stated but definitely worth pondering!